docs/en/FAQ/Load-outside-jar-to-classpath.md

```diff
@@ -0,0 +1,24 @@
+# Load outside jar to classpath 
+ 
+### Problem
+In some cases, especially when the target java app is packaged in an old way,that mean the depency libs directory is out of the target application jar. We need to add some VM options to make sure the java applications starts well, like "java -cp **/lib -ext.dirs **/lib" .
+But when we add these VM options, the target ClassLoaders may not be able to load any classes from skywalking-agent.jar cause it's out of the path i have defined previously. The target app classloader or any of its parent classloader will load the class with failure and push it down to AgentClassLoader to load the class ,and finally ,the not class found exception will stop the application from start.
```

Comment:
```suggestion
But when we use `--ext.dirs` options, the target ClassLoader(s) may not be able to load any classes from skywalking-agent.jar due to your codes are loaded by Extension Class Loader, but the agent class is going to be loaded in Application Class Loader as it is a Java agent. 
SkyWalking agents provides native ways to support Bootstrap Class Loader to avoid similar issue, but doesn't support this case.
```